notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/InstallationTemplate.java

```diff
@@ -97,6 +102,69 @@ public void removeHeader(String name) {
         mHeaders.remove(name);
     }
 
+    /**
+     * Serialize InstallationTemplate to JSONObject.
+     *
+     * @param name Name of template
+     * @param installationTemplate The templates that should no longer be in the collection.
+     * @return serialized templateObject.
+     */
+    public static JSONObject serialize(String name, InstallationTemplate installationTemplate) {
```

Comment:
Let's make this package-private. I don't want to leak public surface area. :)

Part of it too is that at the moment there's only the one JSON serialization form that is important, but in the future we could conceivably need to serialize this differently for different scenarios. For instance, API-Versions could require different formats, or we could even want to store the template differently in Shared preferences than we do when it hits the wire.

Reply:
fixed